tests/foreman/ui/test_subscription.py

```diff
@@ -331,3 +337,91 @@ def test_positive_view_VDC_subscription_products(self):
                     VDC_SUBSCRIPTION_NAME)
                 self.assertEqual(len(content_products), 1)
                 self.assertIn(vds_product_name, content_products)
+
+    @skip_if_not_set('fake_manifest')
+    @tier3
+    def test_positive_view_VDC_guest_subscription_products(self):
+        """Ensure that Virtual Data Centers guest subscription Provided
+        Products and Content Products are not empty.
+
+        :id: 4a6f6933-8e26-4c47-b544-a300e11a8454
+
+        :steps:
+            1. Upload a manifest with Virtual Datacenters subscription
+            2. Config a virtual machine virt-who service for a hypervisor
+            3. Ensure virt-who hypervisor host exist
+            4. Attach Virtual Datacenters subscription to the virt-who
+               hypervisor
+            5. Go to Content -> Red Hat Subscription
+            6. Select Virtual Datacenters subscription with type Guests of
+               virt-who hypervisor
+
+        :expectedresults:
+            1. The Virtual Data Centers guests subscription Provided Products
```

Comment:
You verify not only the fact that list is not empty, but making direct assertions for product names